0023-zarf-named-configs/zep.yaml

```diff
@@ -0,0 +1,29 @@
+schema-version: 1.0.0
+
+title: Zarf Named Configs
+zep-number: 0023
+authors:
+  - "@racer159"
+status: implementable
```

Comment:
Let's nudge this back to `provisional` until we converge on a few more implementation details. Ideally we can prove this out with an experimental branch first, but that's not necessary in all cases.